src/commands/entities/configure.ts

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import chalk from 'chalk'
+import {flags} from '@oclif/command'
+import makeDebug from 'debug'
+
+import * as EntitiesAPI from '../../mix/api/entities'
+import * as MixFlags from '../../utils/flags'
+import {EntitiesConfigureParams, MixClient, MixResponse, MixResult} from '../../mix/types'
+import MixCommand from '../../utils/base/mix-command'
+import {DomainOption} from '../../utils/validations'
+import {validateRegexEntityParams, validateRuleBasedEntityParams} from '../../utils/validations'
+
+const debug = makeDebug('mix:commands:entities:configure')
+
+export default class EntitiesConfigure extends MixCommand {
+  static description = `configure an entity
+
+Use this command to configure an entity in a project.
+
+NOTE: This command cannot change the type of an entity.
+When using this command, you must provide the current type
+of the entity. Use command entities:convert to change the
```

Comment:
Should we include this line ahead of `entities:convert` existing? Or is it coming out in the same release as this PR